contents/teams/feature-success/objectives.mdx

```diff
@@ -2,25 +2,16 @@ As always, reliability is the #1 unwritten goal: Making sure feature flags are r
 
 ### Objective: Make sure feature flags can handle 10x current scale
 
-Last quarter we hit some scaling limits on flags: It's now very expensive to run flags on Django and we'll hit some scaling limits at 5x our scale.
+Last to last quarter we hit some scaling limits on flags: It's now very expensive to run flags on Django and we'll hit some scaling limits at 5x our scale.
 
-To get ahead of this problem, we'll rewrite our flags service to be more performant and reliable.
+To get ahead of this problem, we'll rewrite our flags service to be more performant and reliable. This is a continuation of last quarter's goal, since we aren't done yet.
 
-### Objective: Polish new experiments UI & collect feedback
+### Objective: No-code experiments 
```

Comment:
I think we should split no-code experiments and "Split out experiments into its own product" into two separate goals.